astro/src/content/docs/lifecycle/migrate-users/offboard.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: "Offboard: Leaving FusionAuth"
+description: Learn how to export your data from FusionAuth to migrate to another authentication service.
+section: lifecycle
+subcategory: migrate users
+---
+
+import Aside from 'src/components/Aside.astro';
+import IconButton from 'src/components/IconButton.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineField from 'src/components/InlineField.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import Diagram1 from 'src/diagrams/docs/lifecycle/migrate-users/offboardDiagram.astro';
+
+## Introduction
+
+This guide explains how to export user data from FusionAuth to prepare it for migration to another system. This information may be useful for you to:
+- Export user data to a data warehouse for analysis.
+- Assess how easy migrating from FusionAuth would be before signing up.
+- Switch from FusionAuth to another authentication service.
+
+You can follow this guide to try some examples of exporting data. You will start a new FusionAuth instance with a sample user and run SQL queries. If you already use FusionAuth, you can export your data from your existing instance.
+
+To learn about authentication migration in general, please see the [generic migration guide](/docs/lifecycle/migrate-users/genericmigration) and the [migration overview](/docs/lifecycle/migrate-users). The principles in these guides that explain how to migrate **to** FusionAuth can also be used to migrate **from** FusionAuth to another service. It is important to understand the different migration strategies, especially online and offline migrations (static versus dynamic migrations).
```

Comment:
I might be off on this if there is another PR waiting review, but should this be https://fusionauth.io/docs/lifecycle/migrate-users/general-migration?

```suggestion
To learn about authentication migration in general, please see the [generic migration guide](/docs/lifecycle/migrate-users/general-migration) and the [migration overview](/docs/lifecycle/migrate-users). The principles in these guides that explain how to migrate **to** FusionAuth can also be used to migrate **from** FusionAuth to another service. It is important to understand the different migration strategies, especially online and offline migrations (static versus dynamic migrations).
```

Reply:
@alex-fusionauth  - yeah the intention here was to link to https://github.com/FusionAuth/fusionauth-site/pull/2999/

but I think it still makes sense to link to the one that's live already so let's do that for now to not block this one. 

We might also need to rethink the naming as it's a bit confusing to have 'general' and 'generic'. There's a comment in the other one about maybe renaming it to 'home-grown' - any opinions on this?

Reply:
Completely understand. In my opinion, I don't love homegrown as it takes away somehow from this solution. I think a generally accepted thing vs. generic is fine.

I almost equate this to a facial tissue.

Kleenex is the generally accepted term for facial tissue.
However, the generic version is Facial Tissue. 

Happy to update this link once we get #2999 all the way down the track as well.